exo-go/src/application/tester.go

```diff
@@ -0,0 +1,79 @@
+package application
+
+import (
+	"fmt"
+
+	"github.com/Originate/exosphere/exo-go/src/config"
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+// Tester runs tests for all internal services of the application
+type Tester struct {
+	AppConfig              types.AppConfig
+	InternalServiceConfigs map[string]types.ServiceConfig
+	ServiceData            map[string]types.ServiceData
+	AppDir                 string
+	homeDir                string
+	Logger                 *Logger
+	logChannel             chan string
+}
+
+// NewTester is Tester's constructor
+func NewTester(appConfig types.AppConfig, logger *Logger, appDir, homeDir string) (*Tester, error) {
+	internalServiceConfigs, err := config.GetInternalServiceConfigs(appDir, appConfig)
+	if err != nil {
+		return &Tester{}, err
+	}
+	return &Tester{
+		AppConfig:              appConfig,
+		InternalServiceConfigs: internalServiceConfigs,
+		ServiceData:            config.GetServiceData(appConfig.Services),
+		AppDir:                 appDir,
+		homeDir:                homeDir,
+		Logger:                 logger,
+		logChannel:             logger.GetLogChannel("exo-test"),
```

Comment:
I believe if we stop overriding the log channels of created instances of runner and initializer we won't need this anymore and can just use the logger directly.